openkfw/TruBudget
# 2103 api: configurable refresh token expiration

### Checklist




### Description


### How to test

Set refresh and access token expiration to a short time, in hours or fractions of hours. User should be forcibly logged out after a refresh token expires.
REFRESH_TOKEN_EXPIRATION=0.1
ACCESS_TOKEN_EXPIRATION=0.05




Closes #2103
